### DISCUSSION TITLE
Feature Request: Allow EXPORT DATABASE to preserve comments and only export schema.sql

### DISCUSSION THREAD

**Discussion by zaneselvans** [duckdb/duckdb#13178] [Ideas]
In order to facilitate building documentation for our database programmatically, it would be very helpful if `EXPORT DATABASE` had the ability to include appropriate `COMMENT ON` statements for the tables and columns being created, which would pull the content of comments from the database being exported.

It would also be helpful if there were an option to export only the `schema.sql` file -- since for documentation purposes we don't need any of the data, just the description of the database structure, and the parquet or CSV file outputs can take up quite a bit of disk space.

See also https://github.com/sqlalchemy/sqlalchemy/discussions/11656

**Comment by Tishj** [comment]
https://duckdb.org/docs/sql/statements/copy#copy-from-database--to already supports only copying schema, this can likely be extended to export database
